playbooks/roles/nginx/templates/assets.j2

```diff
@@ -10,6 +10,12 @@ server {
     set $communications_s3_url "{{ communications_s3_url }}";
     set $submissions_s3_url "{{ submissions_s3_url }}";
 
+    # host must be unset for S3 requests
+    proxy_set_header Host "";
```

Comment:
Shouldn't this be the default `$proxy_host`? Or can we drop this line altogether? (I'm not sure if nginx will use default headers when all the proxy_set_header inheritance is involved)